src/wasm-lib/kcl/src/std/shapes.rs

```diff
@@ -66,28 +84,71 @@ async fn inner_circle(
         SketchSurfaceOrGroup::SketchSurface(surface) => surface,
         SketchSurfaceOrGroup::SketchGroup(group) => group.on,
     };
-    let mut sketch_group = crate::std::sketch::inner_start_profile_at(
-        [center[0] + radius, center[1]],
+    let sketch_group = crate::std::sketch::inner_start_profile_at(
+        [data.center[0] + data.radius, data.center[1]],
         sketch_surface,
         None,
         exec_state,
         args.clone(),
     )
     .await?;
 
-    // Call arc.
-    sketch_group = crate::std::sketch::inner_arc(
-        crate::std::sketch::ArcData::AnglesAndRadius {
-            angle_start: 0.0,
-            angle_end: 360.0,
-            radius,
+    let angle_start = Angle::from_degrees(0.0);
+    let angle_end = Angle::from_degrees(360.0);
+
+    if angle_start == angle_end {
```

Comment:
You don't need this, clearly angle_start and angle_end are constants and don't need to be checked for equality IMO

Reply:
thank you, copy paste error from https://github.com/KittyCAD/modeling-app/blob/dabf256e2bfcede1f36a48b0a73069c46cdc2822/src/wasm-lib/kcl/src/std/sketch.rs#L1583

But yeah doesn't make sense here.